.buildkite/pipeline.yml

```diff
@@ -7,8 +7,7 @@ steps:
     agents:
       queue: project42
     timeout_in_minutes: 60
-  - wait
```

Comment:
is it expected to have lost the wait while commenting out the erc20 tests command ?

Reply:
The wait was added by me in an attempt to limit the impact of the ERC20 tests which was causing a weird bug with buildkite fetching from the wrong repository. But it was more like a workaround and was removed in #1086 (a real fix for that specific issue).